tmt/steps/execute/__init__.py

```diff
@@ -205,33 +219,38 @@ def base_command(cls, method_class=None, usage=None):
         @click.option(
             '-h', '--how', metavar='METHOD',
             help='Use specified method for test execution.')
-        def execute(context, **kwargs):
+        def execute(context: Any, **kwargs: Any) -> None:
             context.obj.steps.add('execute')
             Execute._save_context(context)
 
         return execute
 
     @classmethod
-    def options(cls, how=None):
+    def options(cls, how: Optional[str] = None) -> List[click.Option]:
```

Comment:
As discussed in [#1319](https://github.com/teemtee/tmt/pull/1319#discussion_r917014429), this should return `List[ClickOptionDecoratorType]` which is being added in #1304. So blocking this PR until #1304 is merged.

Reply:
Resolved.